### DISCUSSION TITLE
Cannot find Settings > Images Tab

### DISCUSSION THREAD

**Discussion by machine1111-code** [open-webui/open-webui#7527] [Issues]
Hello,

I am trying to locate the Images Tab under Settings.
Anyone know why it is not there? How do I fix it?

I tried to troubleshoot by installing and uninstalling different versions listed under <https://docs.openwebui.com/features/images/> and <https://docs.openwebui.com/getting-started/quick-start/>, but neither shows the tab successfully:
"docker run -d -p 3000:8080 --add-host=host.docker.internal:host-gateway -e AUTOMATIC1111_BASE_URL=http://host.docker.internal:7860/ -e ENABLE_IMAGE_GENERATION=True -v open-webui:/app/backend/data --name open-webui --restart always ghcr.io/open-webui/open-webui:main"

![Upload](https://github.com/user-attachments/assets/fd04678b-8a57-4563-96d2-0651afe7d07f)

Thank you all in advance.

**Comment by Peter-De-Ath** [comment]
It is under Admin Settings -> Settings -> Images

**Reply by machine1111-code** [reply]
![Upload2](https://github.com/user-attachments/assets/23a6955d-b3ac-4d0d-9e25-9af238680dbf)

Also missing under Admin Settings.

**Reply by machine1111-code** [reply]
All it took was enlarging of the browser. A hidden vertical menu surfaced.

Thank you for the help
